airbyte-integrations/bases/base-normalization/build.gradle

```diff
@@ -10,3 +10,5 @@ airbytePython {
 dependencies {
     implementation files(project(':airbyte-integrations:bases:airbyte-protocol').airbyteDocker.outputs)
 }
+
+installReqs.dependsOn(":airbyte-integrations:bases:airbyte-protocol:installReqs")
```

Comment:
how did you think about picking :installReqs versus just depending on the build of that module? i'm wondering if we are going to make a bad assumption in the future. i'm also fine with keeping this though if you think it's the right way.

Reply:
This should go away in the future, and it may go away in favor of the build step. However, the build step in the future might not match what we have now, so I think this is the lightest weight shim to tide us over.